lib/honeybadger/config.js

```diff
@@ -0,0 +1,46 @@
+import Honeybadger from "@honeybadger-io/js";
+
+const setupHoneyBadger = () => {
+  // https://docs.honeybadger.io/lib/javascript/reference/configuration.html
+  const sharedHoneybadgerConfig = {
+    apiKey: process.env.HONEYBADGER_API_KEY,
+    environment: process.env.NODE_ENV,
+    projectRoot: "webpack://_N_E/./",
+    revision: process.env.HONEYBADGER_REVISION,
```

Comment:
The way to get the revision differs in Amplify than it did when building under GitHub Actions. I believe it should be
```suggestion
    revision: process.env.AWS_COMMIT_ID,
```